### PR TITLE
6pm-4 qa heroku deployment

### PR DESCRIPTION
**Pull request by ljvcs**
I created the qa deployment for 6pm-4 on heroku, and included links to the running app and dashboard in the README. The app is deployed to the heroku qa deployment.

### CODE REVIEW

**Review by adrianlindell — APPROVED** [review]
LGTM 🐙

**Comment by pconrad** [comment]
> LGTM 🐙

Please see: https://ucsb-cs156.github.io/topics/code_reviews_github/

**Review by adrianlindell — APPROVED** [review]
LGTM 🐛

**Review by adrianlindell — DISMISSED** [review]

**Review by adrianlindell — APPROVED** [review]
LGTM 🙂

File: README.md [hunk 1]
```diff
@@ -15,7 +15,8 @@
 * [QA Dashboard](https://dashboard.heroku.com/apps/s22-6pm-3-happycows-qa)
 
 6pm-4:
-* [QA](https://team04-w22-6pm-4-qa.herokuapp.com/)
+* [QA](https://s22-6pm-4-happycows-qa.herokuapp.com/)
+* [QA-Dashboard](https://dashboard.heroku.com/apps/s22-6pm-4-happycows-qa)
```
Comment (pconrad) [inline README.md:19]: Part of this task was to give staff access to this dashboard, but that hasn't happened yet:

image

**Review by pconrad — CHANGES_REQUESTED** [review]
Please give staff access to the https://dashboard.heroku.com/apps/s22-6pm-4-happycows-qa deployment; that was part of this issue.

File: README.md (hunk 1 above)
Comment (ljvcs) [reply README.md:19]: Fixed now - staff emails have been added as collaborators to the qa deployment.

**Review by adrianlindell — APPROVED** [review]
Messaged Luke & updated issue acceptance criteria to include "team and staff have access...". LGTM.🎃

**Review by pconrad — APPROVED** [review]

**Pull request merged**